gulpfile.js

```diff
@@ -86,7 +86,7 @@ function write(streams, name) {
   return concat(function(body) {
     return file(dest, body, {src: true})
       .pipe(rev())
-      .pipe(gulpif(production, uglify()))
+      .pipe(uglify())
```

Comment:
Yeah seems like something is wrong. When I set that variable locally, it gets uglified. Can you remove it from this and we can deal with that issue separately?